plugins/woocommerce-blocks/package.json

```diff
@@ -74,9 +74,11 @@
 		"reformat-files": "prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json,ts,tsx}\"",
 		"rimraf": "./node_modules/rimraf/bin.js",
 		"start": "rimraf build/* && cross-env BABEL_ENV=default CHECK_CIRCULAR_DEPS=true webpack --watch",
-		"storybook": "storybook dev  -c ./storybook -p 6006 --ci",
-		"storybook:build": "BABEL_ENV=development storybook build  -c ./storybook -o ./storybook/dist",
+		"storybook": "pnpm --if-present '/^watch:build:project:storybook$/'",
+		"storybook:build": "pnpm --if-present '/^build:project:storybook$/'",
```

Comment:
I'm _pretty sure_ that the regular expression here doesn't do anything anymore, no? The `--if-present` option also isn't necessary since it's just calling another script. Just `pnpm watch:build:project:storybook` is sufficient.

Reply:
Sounds good, removed as part of: https://github.com/woocommerce/woocommerce/pull/51701/commits/6489e5f0ce19371b785d7c2574cd06c5e8bc5b28